src/domain/registration/welcome/welcome.module.scss

```diff
@@ -0,0 +1,31 @@
+@import 'variables';
+@import 'colors';
+@import 'layout';
+
+.welcome {
+    display: grid;
+    text-align: center;
+    .hero {
+        display: grid;
+        @include respond-above(sm) {
+            grid-template-columns: 1fr 10fr 1fr;
+            .heroContainer {
+                grid-column: 2;
+            }
+        }
+     }
+    .submitButton {
```

Comment:
Reuse this style from homeForm submit button

Reply:
Like this?